wrangler.example.toml

```diff
@@ -1,9 +1,8 @@
 name = "stateless-discord-bot"
-type = "rust"
 account_id = "{{ACCOUNT_ID}}"
 workers_dev = true
-route = ""
-zone_id = ""
```

Comment:
No longer required, as CloudFlare detects them automatically.

```
    - 😶 Ignored: "type":
      Most common features now work out of the box with wrangler, including modules, jsx,
  typescript, etc. If you need anything more, use a custom build.
    - Deprecation: "zone_id":
      This is unnecessary since we can deduce this from routes directly.
    - The "route" field in your configuration is an empty string and will be ignored.
      Please remove the "route" field from your configuration.
```